lil_aretomo/utils.py

```diff
@@ -47,11 +47,12 @@ def get_aretomo_command(
         nominal_tilt_axis_angle: Optional[float] = None,
         do_local_alignments: bool = True,
         n_patches_xy: Optional[Tuple[int, int]] = None,
-        gpu_ids: Optional[Sequence[int]] = None
+        gpu_ids: Optional[Sequence[int]] = None,
+        executable: Optional[str] = 'AreTomo',
 ) -> List[str]:
     """Generate a command which can be used to run AreTomo."""
     command = [
-        'AreTomo',
+        executable,
```

Comment:
can you change to: 

`f'{executable}'`

please?

Reply:
done